### DISCUSSION TITLE
Local Instance requires Anvil import

### DISCUSSION THREAD

**Discussion by gokalpcelik** [broadinstitute/seqr#2924] [Q&A] [answered]
Hi

I was using SEQR on my local server however latest seqr docker image update killed the ability to import elastic search imports directly. 

I cannot roll back to a previous image without knowing which tag had no such requirement. What should I do with this now? Can you help me about this issue?

Thanks. 

![image](https://user-images.githubusercontent.com/37572619/186952918-dc0d3ca6-f783-4a42-89e5-0a36555e1268.png)

EDIT: looking at the source code for ui components it looks like this might have been an unintentional error where dataset vs additional data modal is selected depending on the user permissions. I am already the admin user and I should be able to see the dataset (Edit Dataset link) instead of just the Load Additional Data modal. I will be modifying the code accordingly.

**Comment by karynne7** [comment]
I have not updated our version of Seqr, but I could see this being a big problem for us, too.

**Comment by gokalpcelik** [comment]
Here is the problematic section.

```
const EditDatasetsButton = React.memo(({ project, user }) => {
  const showLoadWorkspaceData = !project?.isAnalystProject && project?.canEdit
  return (
    (user.isDataManager || user.isPm || showLoadWorkspaceData) ? (
      <Modal
        modalName={MODAL_NAME}
        title={showLoadWorkspaceData ? 'Load Additional Data From AnVIL Workspace' : 'Datasets'}
        size="small"
        trigger={<ButtonLink>{showLoadWorkspaceData ? 'Load Additional Data' : 'Edit Datasets'}</ButtonLink>}
      >
        {showLoadWorkspaceData ? (
          <AddWorkspaceDataForm
            params={project}
            successMessage="Your request to load data has been submitted. Loading data from AnVIL to seqr is a slow process, and generally takes a week. You will receive an email letting you know once your new data is available."
          />
        ) : (
          <Tab panes={user.isDataManager ? PANES : IGV_ONLY_PANES} />
        )}
      </Modal>
    ) : null
  )
})
```

And this was a former code

```
const EditDatasetsButton = React.memo(({ user }) => (
  (user.isDataManager || user.isPm) ? (
    <Modal
      modalName={MODAL_NAME}
      title="Datasets"
      size="small"
      trigger={<ButtonLink>Edit Datasets</ButtonLink>}
    >
      <Tab panes={user.isDataManager ? PANES : IGV_ONLY_PANES} />
    </Modal>
  ) : null
))



```
I might revert back to this build directly or I will modify the gcloud-prod docker image manually I guess.

**Comment by hanars** [comment, marked as answer]
Yes this is a bug, thanks for reporting

**Comment by hanars** [comment]
Work to fix this is being tracked in this issue: https://github.com/broadinstitute/seqr/issues/2925

**Comment by hanars** [comment]
This should be fixed in the latest seqr release, please update your docker image and let me know if that fixes the issue

**Reply by gokalpcelik** [reply]
Yep. It is fixed. Thanks for the help.
